### PR TITLE
Separate ingress egress

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Description
Separate ingress egress

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

File: openapi/kas-fleet-manager.yaml [hunk 1]
```diff
@@ -1004,9 +1004,9 @@ components:
               type: string
             size_id:
               type: string
-            ingress_throughput_per_sec:
+            ingress_per_sec:
               type: string
-            egress_throughput_per_sec:
+            egress_per_sec:
```
Comment (machi1990) [inline openapi/kas-fleet-manager.yaml:1009]: Are these changes intentional? If so, let's make the UI/CLI team aware of it

Comment (pawelpaszki) [reply openapi/kas-fleet-manager.yaml:1009]: that's what I understood from the chat. I am happy to reverse it

Comment (pawelpaszki) [reply openapi/kas-fleet-manager.yaml:1009]: ok. makes sense

Comment (pb82) [reply openapi/kas-fleet-manager.yaml:1009]: @pawelpaszki sorry, yes I should have been more clear. The goal is to update the names for the data plane (capacity struct) only. Unfortunately the contract with the UI now uses different names, but revising that should be a separate task.

Comment (pawelpaszki) [reply openapi/kas-fleet-manager.yaml:1009]: opened new PR here instead of this one: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/946
